### PR TITLE
feat(incorrect-information-button): Information button about how to change your data

### PR DESCRIPTION
**Pull request by Zeroks77**
Hey guys,

as promised here my first pr handling the Issue# #174  .
Please review my code and tell me what I can do in a better way.
I'm looking forward to help to improve this project with you :)

Btw I'm sorry that my editor reorderd some code ^^ i didnt mean to create that many of work for you

### CODE REVIEW

**Comment by mschwrdtnr** [comment]
Hey @Zeroks77 thank you for your first PR. You're helping to make this project bigger and ship a better software! 🎉 

It looks like that you didn't used our workspace prettier settings. This is the reason why the whole code is different formatted. We are using `prettier.printWidth": 120` 
**Can you please format the code like our settings and push your changes again?**

File: Phonebook.Frontend/src/app/shared/dialogs/userinformation/dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Component, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material';
+import { DialogData } from 'src/app/shared/components/user/user-detail/user-detail.component';
+
+@Component({
+  selector: 'app-dialog',
+  templateUrl: './dialog.component.html',
+  styleUrls: ['./dialog.component.scss']
+})
+export class DialogComponent {
```
Comment (paule96) [inline Phonebook.Frontend/src/app/shared/dialogs/userinformation/dialog.component.ts]: rename this to something like userinformationdialog

**Comment by DanielHabenicht** [comment]
https://dev.azure.com/T-Systems-MMS/phonebook/_build/results?buildId=1515&view=logs&j=93f9ac11-2734-591d-619b-c9f557d843a2&t=153673d6-90f0-53b6-fdd2-01489a0d5bc4&l=1
The preview build is failing because of incorrect credentials...

**Pull request merged**